Gruntfile.js

```diff
@@ -0,0 +1,89 @@
+module.exports = function(grunt) {
+	grunt.initConfig({
+		pkg: grunt.file.readJSON('package.json'),
+
+		clean: {
+			dist: ['dist']
+		},
+
+		copy: {
+			main: {
+				expand: true,
+				flatten: true,
+				src: 'src/headless.esm.js',
+				dest: 'dist/',
+			},
+		},
+
+		uglify: {
+			options: {
+				banner: '/*!\n * Canada.ca Search UI Connector / Connecteur IU de Recherche pour Canada.ca\n' + 
+				' * v<%= pkg.version %> - ' + '<%= grunt.template.today("yyyy-mm-dd") %>\n*/'
+			},
+
+			dist: {
+				files: {
+					'dist/connector.min.js': ['src/connector.js']
+				}
+			}
+		},
+
+		cssmin: {
+			dist: {
+				files: {
+					'dist/connector.min.css': ['src/connector.css']
```

Comment:
I understand and already aware of your point. The styling is for the Type-ahead (query suggestion) box only, which is not something currently available in GCWeb and, as I mentioned earlier, can be vetted to be added to GCWeb at a later date. This will serve the purpose of an add-on feature on the Basic search page.